### PR TITLE
misc tidy

### PR DESCRIPTION
**Pull request by casperdcl**

### CODE REVIEW

File: pretrained_model_tuner.py [hunk 1]
```diff
@@ -1,51 +1,51 @@
-from __future__ import print_function
-from __future__ import division
```
Comment (casperdcl) [inline pretrained_model_tuner.py:2]: `dvclive` only works in py3 so no need for this py2 legacy code

File: pretrained_model_tuner.py [hunk 2]
```diff
@@ -93,60 +93,55 @@ def train_model(model, dataloaders, criterion, optimizer, num_epochs=2, is_incep
                 dvclive.log('val_loss', epoch_loss)
 
                 val_acc_history.append(epoch_acc)
-                
-                dvclive.next_step()
```
Comment (casperdcl) [inline pretrained_model_tuner.py:97]: seems better to move `next_step()` below to avoid confusion

File: pretrained_model_tuner.py [hunk 3]
```diff
@@ -1,51 +1,51 @@
...
 
     val_acc_history = []
 
     best_model_wts = copy.deepcopy(model.state_dict())
     best_acc = 0.0
 
-    for epoch in range(num_epochs):
-        print('Epoch {}/{}'.format(epoch, num_epochs - 1))
-        print('-' * 10)
+    for epoch in range(dvclive.get_step(), dvclive.get_step() + num_epochs):
```
Comment (casperdcl) [inline pretrained_model_tuner.py:39]: better epoch reporting (for resuming training from last epoch)

File: requirements.txt [hunk 4]
```diff
@@ -1,5 +1,6 @@
 dvc
+dvclive
```
Comment (casperdcl) [inline requirements.txt:2]: was missing dependency

**Review by casperdcl — COMMENTED** [review]
explanation of changes

File: pretrained_model_tuner.py [hunk 5]
```diff
@@ -1,51 +1,51 @@
...
+        print(f"Epoch {epoch}/{num_epochs - 1}")
```
Comment (casperdcl) [inline pretrained_model_tuner.py:40]:
```suggestion
    total = dvclive.get_step() + num_epochs
    for epoch in range(dvclive.get_step(), total):
        print(f"Epoch {epoch}/{total - 1}")
```

**Review by flippedcoder — APPROVED** [review]

**Pull request merged**